app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.test.tsx

```diff
@@ -68,7 +68,7 @@ describe("PageNew", () => {
       const {
         history: { navigate },
       } = renderWithRouter(
-        <MockedCache mocks={[mock, mock]}>
+        <MockedCache mocks={[mock, mock, mock]}>
```

Comment:
We need an additional mock here because `/my-special-slug/`, which is hit in this test, now needs a mock.